torch/_inductor/pattern_matcher.py

```diff
@@ -571,15 +571,19 @@ def pytree_flatten(
     ) -> Tuple[Sequence[Any], Union[_SimpleSpec, pytree.TreeSpec]]:
         type_mapping = {immutable_list: tuple, list: tuple, immutable_dict: dict}
 
-        def replace_type(x: Any) -> Any:
+        def convert_type(x: Any) -> Any:
```

Comment:
It seems something is broken with `ghstack`. This change was introduced in the previous PR in the stack, but now it shows up in every PR in the stack. I tried to use a `reword` rebase to change the orig commits in the stack and use `ghstack --force` to force push the commits. But that doesn't work. cc @ezyang.